lib/resources/example_json/r4_examples_coverage.rb

```diff
@@ -373,6 +373,66 @@ module Resources
       ]
     }.freeze
 
+    R4_COVERAGE_PATIENT_SOCIAL_HEALTHCARE_CREATE ||= {
```

Comment:
Are we not requiring 'subscriber' for public healthcare writes? We should note that in the body fields since we say it is required in patient level writes.

Reply:
Added multiple requirement notes: https://github.com/cerner/fhir.cerner.com/pull/676/commits/e9e4172ecc18c83453561551caa7ed78cc3004ef